linera-views/src/common.rs

```diff
@@ -539,33 +539,33 @@ pub trait Context {
 }
 
 /// Implementation of the [`Context`] trait on top of a DB client implementing
-/// [`KeyValueStoreClient`].
+/// [`KeyValueStore`].
 #[derive(Debug, Default, Clone)]
-pub struct ContextFromDb<E, DB> {
+pub struct ContextFromStore<E, S> {
```

Comment:
The trait could be called `ViewContext` and this struct just `Context`.

Reply:
(next PR ok)

Reply:
Yes, next PR though I am unsure about that change.